packages/leancode_lint/test/lints_test_app/lib/bloc_related_classes_equatable_test.dart

```diff
@@ -0,0 +1,84 @@
+// for tests
+// ignore_for_file: prefer_equatable_mixin, bloc_const_constructors
+
+import 'package:bloc_presentation/bloc_presentation.dart';
+import 'package:equatable/equatable.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+// Bloc-related classes that are not `equatable` are flagged
+class Test1Bloc extends Bloc<Test1Event, Test1State>
+    with BlocPresentationMixin<Test1State, Test1PresentationEvent> {
+  Test1Bloc() : super(Test1State());
+}
+
+final class
+// expect_lint: bloc_related_classes_equatable
+Test1Event {}
+
+final class
+// expect_lint: bloc_related_classes_equatable
+Test1State {}
+
+final class
+// expect_lint: bloc_related_classes_equatable
+Test1PresentationEvent {}
+
+//////////////////////////////////////////////////////////////////////////
+
+// Bloc-related classes that `extend Equatable` are not flagged
```

Comment:
I think it would be better if we flag them also. Using Equatable is more problematic than using a mixin version. Maybe we could do such a flag with `extends` equatable as an info severity instead of a warning? I wonder if there is any case where `extends Equatable` is a considerably more suitable solution.

Reply:
Yes, I've seen it later, unfortunately. Good work :)